packages/client/src/views/SearchResult/SearchResult.tsx

```diff
@@ -313,6 +315,7 @@ class SearchResultView extends React.Component<
               !declarationIsRegistered &&
               !declarationIsCertified &&
               !declarationIsArchived &&
+              !declarationIsCorrectionRequested &&
```

Comment:
Could you clean up this conditional here a little bit please? Just by reading this I have a very hard time understanding when exactly does this conditional apply. I see you added a check that the record is not in "correction requested" state but then on line 328 there's code to handle that

Reply:
@rikukissa could you please have a look?  [6dc8606](https://github.com/opencrvs/opencrvs-core/pull/7226/commits/6dc86068854c56712903346549043ef54cc3e386)